### PR TITLE
Motor controller

### PR DESCRIPTION
**Pull request by stephen0517**

### CODE REVIEW

**Comment by John-Carr** [comment]
This is a merge of a forked project so please do not merge yet.

File: MotorController/mcProgramming [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+##  **Programming:**  
+
+1. Use file access options to either save a controller configuration or to load a previously saved controller configuration
+2. Use controller access to set the COM port you previously identified. This section also contains the write and read options. 
+	1. To load the settings that are currently on the controller use the read option. 
+	2. To upload a new controller congifuration use the write option. Be warned, writing to the Mitsuba does take a while (sometime a couple minutes)
+	3. Since writing takes a long time, make sure no one turns off the car so you dont have to redo the writing process from the start
+3. Click End and close the tool when writing is completed   
+
+##  **Options:**
```
Comment (John-Carr) [inline MotorController/mcProgramming:19]: Should we leave a note to where the documentation?

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,3 @@
 _build/*
 .vscode/*
+MotorController/motor_controller_programming.txt
```
Comment (John-Carr) [inline .gitignore:3]: Should this be md?

File: MotorController/index.rst [hunk 3]
```diff
@@ -1,4 +1,10 @@
 Motor Controller
 ====
 
-Stephen better put stuff here
+The Motor Controller Subsystem is responsible for understanding how to wire the controller to the motor as well as interface with the controller through the configuration tool. In addition, this system develops a PCB that supplements the controller by adding features not supported by the controller.
+
+.. toctree::
+   :maxdepth: 1
```
Comment (John-Carr) [inline MotorController/index.rst:7]: doesn't look like this is showing up in the build

**Review by John-Carr — CHANGES_REQUESTED** [review]
looks like we have 2 mcprogramming files and this is a forked project. We need to merge this into the main repo then have @stephen0517 reset his local repo to point to this repo rather than his forked one.

File: .gitignore (hunk 2 above)
Comment (Seth10001) [reply .gitignore:3]: Why is this in the ignore?

File: MotorController/index.rst (hunk 3 above)
Comment (Seth10001) [reply MotorController/index.rst:7]: This `index` file needs to be added to the root `index` here: https://github.com/Solar-Gators/docs/blob/master/index.rst

**Review by Seth10001 — CHANGES_REQUESTED** [review]
You should have access to the repo, so in the future make a branch under the repo instead of forking.

The motor controller isn't showing up on the home page or any of the other pages. This is happening because of the main `index.rst`'s toctree has to be edited here: https://github.com/Solar-Gators/docs/blob/master/index.rst

Just add a microcontroller subteam there. Thank you for documenting :)

**Comment by stephen0517** [comment]
This was forked. Made new pull request.

**Pull request closed without merging**